### PR TITLE
Fix timing bug in postroll?.enter

### PR DESCRIPTION
**Pull request by incompl**

### CODE REVIEW

File: test/videojs.ads.test.js [hunk 1]
```diff
@@ -1051,6 +1051,7 @@ QUnit.test('ad impl can notify contrib-ads there is no postroll', function(asser
   this.player.trigger('nopostroll');
   this.player.ads.state = 'content-playback';
   this.player.trigger('contentended');
+  this.clock.tick(5);
```
Comment (misteroneill) [inline test/videojs.ads.test.js]: Is there a reason `tick(1)` wouldn't work? Mostly curious here.

File: src/videojs.ads.js [hunk 2]
```diff
@@ -668,8 +668,8 @@ var
           enter: function() {
             this.snapshot = getPlayerSnapshot(player);
             if (player.ads.nopostroll_) {
-              player.ads.state = 'content-resuming';
               window.setTimeout(function() {
+                player.ads.state = 'content-resuming';
```
Comment (dmlap) [inline src/videojs.ads.js]: Should we note why this is happening in a setTimeout() somewhere? This is definitely in the non-obvious category for me.

File: test/videojs.ads.test.js (hunk 1 above)
Comment (incompl) [inline test/videojs.ads.test.js]: I feel safer adding extra time lol

**Comment by incompl** [comment]
@misteroneill @dmlap LGTYN?

**Comment by dmlap** [comment]
LGTM

**Pull request merged**